dashboard/app/models/sections/section.rb

```diff
@@ -306,8 +306,8 @@ def toggle_hidden_script(script, should_hide)
   # once such a thing exists
   def has_sufficient_discount_code_progress?
     return false if students.length < 10
-    csd2 = Script.get_from_cache('csd2-2018')
```

Comment:
Note that I think this will need to be updated for teachers redeeming codes next school year, so that progress from the 2020 version of these scripts count towards meeting this criteria?

Reply:
Yep - but those scripts don't exist yet, and/or it's possible units 2 and 3 won't be the ones we check for, if the curriculum changes much.  It's okay to just implement for this year right now.